Assets/Scripts/Game Object Definitions/Entity Definitions/Yard.cs

```diff
@@ -39,7 +39,8 @@ protected override void Start()
 
     public static readonly int YardProximitySquared = 75;
     private static float lastPartTakenTime = 0;
-    private static int partsTakenCombo = 0;
+    private static int partsTakenCombo = 0, shardsTakenCombo = 0;
+    public static string entityID;
```

Comment:
Statics can be difficult to deal with if the names aren't descriptive... could you rename entityID to something like "collectingYardID"?